src/bin/bench-tps.rs

```diff
@@ -311,13 +312,20 @@ fn fund_keys(client: &mut ThinClient, source: &Keypair, dests: &[Keypair], token
             assert!(f.1 >= per_unit);
         }
         println!("generating... {}", to_fund.len());
-        let to_fund_txs: Vec<_> = to_fund
-            .par_iter()
-            .map(|(k, m)| Transaction::system_move_many(k, &m, last_id, 0))
-            .collect();
-        println!("transfering... {}", to_fund.len());
-        to_fund_txs.iter().for_each(|tx| {
-            let _ = client.transfer_signed(&tx).expect("transfer");
+        // try to transfer a few at a time with recent last_id
+        to_fund.chunks(10_000).for_each(|chunk| {
+            let last_id = client.get_last_id();
+            let mut to_fund_txs: Vec<_> = chunk
+                .par_iter()
+                .map(|(k, m)| Transaction::system_move_many(k, &m, last_id, 0))
+                .collect();
+            // randomly distributed the failures
+            // which will allow the tests to function
```

Comment:
I'd prefer a more specific comment here.  Perhaps by including details such as: Which tests are now allowed to function?  Why didn't they function before?